ApollosProject/apollos-prototype
# Fixes Content Items back redirecting to Onboarding

## DESCRIPTION

### What does this PR do, or why is it needed?

Under certain conditions, clicking "Back" on a ContentItem would redirect the user to Onboarding, rather than the appropriate tab screen. This is because "Onboarding" was still technically the top of the card stack.

### What design trade-offs/decisions were made?

### How do I test this PR?

1. Logout.
2. Complete onboarding
3. Open a content item
4. Close the content item
- You should be redirected to he screen you viewed the content item from, not "Onboarding".


### What automated tests did you write?

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
